src/providers/hyperbolic/image.ts

```
@@ -0,0 +1,300 @@
+
+  id(): string {
+    return `hyperbolic:image:${this.modelName}`;
+  }
+
+  toString(): string {
+    return `[Hyperbolic Image Provider ${this.modelName}]`;
+  }
+
+  private getApiModelName(): string {
+    const model = HYPERBOLIC_IMAGE_MODELS.find(
+      (m) => m.id === this.modelName || (m.aliases && m.aliases.includes(this.modelName)),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)